exec/cnex/global.c

```diff
@@ -97,6 +107,27 @@ void print(TExecutor *exec)
     pop(exec->stack);
 }
 
+void chr(struct tagTExecutor *exec)
+{
+    Number x = top(exec->stack)->number; pop(exec->stack);
+
+    if (!number_is_integer(x)) {
+        exec->rtl_raise(exec, "ValueRangeException", "chr() argument not an integer", BID_ZERO);
+        return;
+    }
+    if (number_is_negative(x) || number_is_greater(x, number_from_uint32(0x10ffff))) {
+        exec->rtl_raise(exec, "ValueRangeException", "chr() argument out of range 0-0x10ffff", BID_ZERO);
+        return;
+    }
+
+    Cell *r = cell_createStringCell(0);
+    // ToDo: Implement UTF8 strings here!!
+    char ch = (char)number_to_uint32(x);
+    string_appendChars(r->string, (char*)&ch, 1);
```

Comment:
No cast needed. `&ch` is already a `char *`, you don't need to cast it again.

Reply:
Fixed.